src/components/Header/NavigationMenu.tsx

```diff
@@ -0,0 +1,199 @@
+import { spacing } from "@chakra-ui/theme/foundations/spacing"
+import {
+  Button,
+  Drawer,
+  DrawerContent,
+  DrawerOverlay,
+  HStack,
+  Icon,
+  Link,
+  List,
+  ListItem,
+  ListItemProps,
+  StackDivider,
+  StackProps,
+  SystemStyleObject,
+  useDisclosure,
+  VisuallyHidden,
+  VStack,
+} from "@threshold-network/components"
+import { motion } from "framer-motion"
+import { FC, useRef } from "react"
+import { NavLink } from "react-router-dom"
+import useChakraBreakpoint from "../../hooks/useChakraBreakpoint"
+
+const NavigationMenuMobileContainer: FC<StackProps> = (props) => (
+  <VStack
+    divider={<StackDivider />}
+    {...props}
+    spacing={0}
+    position="fixed"
+    inset={0}
+    h="100vh"
+    py={16}
+    bgGradient="radial(circle at bottom right, #0A1616, #090909)"
+    borderLeft="1px solid"
+    borderColor="whiteAlpha.250"
+    alignItems="stretch"
+  />
+)
+
+const NavigationMenuDesktopContainer: FC<StackProps> = (props) => (
+  <HStack
+    {...props}
+    spacing={0}
+    alignSelf="stretch"
+    alignItems="stretch"
+    ml={{
+      lg: `calc(${spacing[16]} - ${spacing[5]})`,
+      xl: `calc(142px - ${spacing[5]})`,
+    }}
+    mr="auto"
+  />
+)
+
+const HamburgerIcon: FC<{ isToggled: boolean }> = ({ isToggled }) => (
+  <Icon
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    w={5}
+    h={5}
+    fill="none"
+    stroke="white"
+    stroke-width={1}
+    stroke-linecap="round"
+    stroke-linejoin="round"
```

Comment:
Should be:

```suggestion
    strokeWidth={1}
    strokeLinecap="round"
    strokeLinejoin="round"
```

The actual implementation produces warnings in the console.

Reply:
Ref commit: 7bb98598